<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectLack</class>
 <widget class="QDialog" name="DialogSelectLack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>691</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>缺货单号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_lackid"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>出货单号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_outid"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>货品号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_goodsid"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnselect">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
